contracts/lootbox/LootboxByItem.sol

```diff
@@ -96,7 +96,7 @@ contract LootboxByItem is ILootbox, ERC1155Upgradeable, AccessControlUpgradeable
 
```

Comment:
For line 75, I think you can just plug dataStorage.getEnabled(_tokenId) into the require statement.